### ISSUE TITLE
Inkscape locks up on export

### ISSUE THREAD

**Issue by tofarley** [tofarley/coa_tools#4]
When I ran the script against an actual character Inkscape locked up repeatedly.

I noticed in the output of `inkscape --query-all` that there were some missing layer IDs, but those layer IDs were visible in the svg file. I ran the "Clean up Document" command inside of Inkscape (File -> Clean up Document) and then I was able to export successfully.

Cleaning the document is probably good practice anyway, but it would probably be a good idea to do some error checking and *at least* abort cleanly if the file has cruft.

**Comment by tofarley** [comment]
It appears that we crash on empty layers too. I need some better error handling to prevent this.

```
$ python inkscape_coa_exporter.py --dpi=90 --path=/home/tofarley/Desktop/ /home/tofarley/Desktop/cat_cleaned.svg
Processing 'left_front_paw'
Processing 'left_front_paw copy'
Processing 'left_rear_paw'
Processing 'tail'
Processing 'body'
Processing 'right_rear_paw'
Processing 'right_rear_paw copy'
Processing 'right_front_leg'
Traceback (most recent call last):
  File "inkscape_coa_exporter.py", line 226, in <module>
    _main()
  File "inkscape_coa_exporter.py", line 222, in _main
    e.affect()
  File "/usr/share/inkscape/extensions/inkex.py", line 268, in affect
    self.effect()
  File "inkscape_coa_exporter.py", line 199, in effect
    pos = [float(img.layers[layerid]['x']), float(img.layers[layerid]['y'])]
KeyError: 'layer4'
tofarley@homeschooled:~/github/coa_tools/Inkscape$ python inkscape_coa_exporter.py --dpi=90 --path=/home/tofarley/Desktop/ /home/tofarley/Desktop/cat_cleaned.svg
```

In this instance layer4 was an empty layer, so --query-all didn't return any results for it.

**Comment by tofarley** [comment]
Fixed in 852c0e4cc40c909179030da81f55d5edee97ee75.
